rust/src/storage/file/reuse_list_file.rs

```diff
@@ -62,6 +62,10 @@ impl ReuseListFile {
         Ok(())
     }
 
+    pub fn set_fronzen_count(&mut self, frozen_count: usize) {
```

Comment:
```suggestion
    pub fn set_frozen_count(&mut self, frozen_count: usize) {
```

Here and in the test name below.